### PR TITLE
Make first column caption configurable

### PR DESCRIPTION
**Pull request by marcelstoer**
This removes the hardcoded "NAME" caption on the first column.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Requires jQuery 1.9.1+, CreateJS 1.0.0.
           },
           // Other options
           option: {
+              // firstColumnCaption: "Scholar"
```
Comment (yn1323) [inline readme.md]: Please make this option NOT commented out

**Comment by yn1323** [comment]
Thanks for pull request.
Made 1 comment. Please check.

**Comment by marcelstoer** [comment]
> Made 1 comment. 

Where? I don't see any comments.

**Comment by yn1323** [comment]
Sorry i haven't  submitted it.

**Pull request merged**